### PR TITLE
WRP-593: Update README.md 

### PR DESCRIPTION
**Pull request by seunghoh**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update README.md

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Added more information about the Enact and useful doc links.
Updated some outdated information in the core and spotlight module

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-593

### Comments

Enact-DCO-1.0-Signed-off-by: Seungho Park <seunghoh.park@lge.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,28 @@
...
+
+# Enact [![Travis](https://img.shields.io/travis/com/enactjs/enact/master?style=flat-square)](https://app.travis-ci.com/github/enactjs/enact) [![npm (scoped)](https://img.shields.io/npm/v/@enact/core.svg?style=flat-square)](https://www.npmjs.com/package/@enact/core) [![license](https://img.shields.io/github/license/enactjs/enact.svg?style=flat-square)](http://www.apache.org/licenses/LICENSE-2.0) [![Gitter](https://img.shields.io/gitter/room/EnactJS/Lobby.svg?style=flat-square)](https://gitter.im/EnactJS/Lobby) [![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen)](https://enactjs.com/docs/developer-guide/contributing)
 
 > A mono-repo containing Enact framework modules
 
+* [core](https://enactjs.com/docs/modules/core/dispatcher) The set of essential building blocks for an Enact-based application.
```
Comment (MikyungKim) [inline README.md:12]: Did you intend to omit `webos`?

File: README.md [hunk 2]
```diff
@@ -1,9 +1,28 @@
-# Enact [![Travis](https://img.shields.io/travis/enactjs/enact.svg?style=flat-square)](https://travis-ci.org/enactjs/enact) [![license](https://img.shields.io/github/license/enactjs/enact.svg?style=flat-square)](http://www.apache.org/licenses/LICENSE-2.0) [![Gitter](https://img.shields.io/gitter/room/EnactJS/Lobby.svg?style=flat-square)](https://gitter.im/EnactJS/Lobby)
+<p align="center">
+  <img src="https://github.com/enactjs/enact/assets/4288375/25a32c12-6a9b-44d4-80b3-a8f82ddcc643">
+</p>
+
+<h1>An app development framework built atop React that’s easy to use, performant and customizable.</h1>
```
Comment (MikyungKim) [inline README.md:5]: 🤔 How about mentioning `Enact` before this description?
```
# Enact [![Travis](https://img.shields.io/travis/com/enactjs/enact/master?style=flat-square)]
```

File: packages/core/README.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ npm install --save @enact/core
 
 ## Test
 
-Tests are implemented in mocha and are run with the Karma runner.
+Unit tests are implemented with Jest. To execute them:
```
Comment (MikyungKim) [inline packages/core/README.md:17]: How about
```suggestion
Unit tests are implemented in Testing Library and are run with Jest. To execute them:
```

File: README.md (hunk 2 above)
Comment (seunghoh) [reply README.md:5]: Do you mean like "Enact is an app development framework...."?

File: README.md (hunk 1 above)
Comment (seunghoh) [reply README.md:12]: Good catch! I just missed it :( Added

File: packages/core/README.md (hunk 3 above)
Comment (seunghoh) [reply packages/core/README.md:17]: I just made the same as what was in the Sandstone, but I prefer your suggestion. I will also update the same section in the Sandstone.

File: README.md (hunk 2 above)
Comment (seunghoh) [reply README.md]: Make sense. Updated!

**Review by MikyungKim — APPROVED** [review]
LGTM

**Pull request merged**